### DISCUSSION TITLE
Can I use a different memory card after the firmware has been flashed and does it stick after a reset?

### DISCUSSION THREAD

**Discussion by Pauligrinder** [alienatedsec/yi-hack-v5#241] [Q&A]
I found an old 2gb memory card in my basement, which I thought I'd use for flashing. I'm normally using a 64gb card in the camera, but I don't want to format it to FAT, so I'm wondering, will it work if I flash it from the small card but then afterwards put the big card in? Does it look for the files after flashing, and if it does for example need the included folder, can I just put that on the bigger card?

And if I need to reset the camera, will the firmware remain?

**Comment by alienatedsec** [comment]
Many questions, simple answers.

You can use 2GB to flash
You need the folder `yi-hack-v5` with the hack files in the root of the `SD Card` at all times. So doesn't matter what card is used after a flash.

**Reply by Pauligrinder** [reply]
Ok, great. So I'll copy all the files onto both the cards, flash using the small one and once it's done pop the bigger one in.
So basically flashing part only needs the FAT32 filesystem, but afterwards the files can be read from Exfat or whatever fs bigger cards use?

I'll give that a go when I get a chance, the camera is sitting between two double windows and there's a lot of decorative stuff on the windowpane, so getting the camera out and putting it back is a bigger operation than the actual "work" 🤣
